.changeset/stale-toys-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@docs/storybook': patch
+---
+
+Move from react-docgen-typescript to react-docgen
```

Comment:
@ryanseddon should we update this to just say we're just overriding the transitive dep in storybook?